src/custom/components/CowSubsidyModal/SubsidyTable.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components/macro'
+import { formatSmartLocaleAware } from 'utils/format'
+import { ClaimTr } from 'pages/Claim/ClaimsTable'
+import { BigNumber } from 'bignumber.js'
+
+const StyledSubsidyTable = styled.table`
+  width: 100%;
+`
+
+const SubsidyTr = styled(ClaimTr)`
+  > th {
+    font-weight: 300;
+  }
+  > td,
+  th {
+    padding: 10px;
+    text-align: center;
+  }
+`
+
+function SubsidyTable({ data }: { data: (string | number)[][] }) {
```

Comment:
if you define the data as just data and not include the headers, is better model and types are also simpler